src/javascripts/controllers/transactionController.js

```diff
@@ -66,27 +67,60 @@ class TransactionController {
 
     addTransaction = () => {
         this.view.transactionView.renderPopup();
-    }
+    };
 
     renderTransactionInfo = async (transactionId) => {
         const transaction = await this.service.transactionService.getTransactionById(transactionId);
         this.view.transactionView.renderPopup(transactionId, transaction);
-    }
+    };
 
     getTransactionById = (id) => {
         this.service.transactionService.getContactById(id);
-    }
-
-
+    };
 
     addSummaryTab = () => {
         const transactions = this.service.transactionService.getTransactions();
         this.view.homeView.renderSummaryDetails(transactions);
-    }
+    };
 
-    addTransactionDetails = () =>{
+    addTransactionDetails = () => {
         this.view.homeView.closeSummaryTab();
+    };
+
+    //----- BUDGET CONTROLLER -----//
+    /**
+     * Initializing the Budget interface and event handlers
+     */
+    async initBudget() {
+        await this.service.budgetService.initBudgetList();
+        this.loadListBudgets();
+        this.view.homeView.addEventRenderBudgetPopup(this.addBudget);
     }
+
+    loadListBudgets = async () => {
+        const budgets = this.service.budgetService.getBudgets();
+        const category = 'Other Income';
```

Comment:
Could you use another name that make sense?

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!